spring/boot2-actuator-autoconfigure/src/test/resources/META-INF/spring.factories

```diff
@@ -0,0 +1,2 @@
+org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
+com.linecorp.armeria.spring.RetryableArmeriaServerGracefulShutdownLifecycleConfiguration
```

Comment:
Question: Don't we need to copy this resource using the Gradle script?

Reply:
Oh, nice idea! 👍